src/fasta-recs.c

```diff
@@ -9,8 +40,63 @@ int main(int argc, char const *argv[])
     }
 
     const char *fasta_fname = argv[1];
-    printf("I should be parsing %s now, but someone forgot to implement that.\n",
-           fasta_fname);
+
+    char *content = readFile(fasta_fname);
+
+    //Parsing into DNA chains
+    int pos = 0;
+    const int length = strlen(content);
```

Comment:
If you decide to change the `calloc` into `malloc` you have to be careful here. This assumes that `content` is zero-terminated, and it won't be if you used `malloc`. Then you would have to terminate it yourself.